source/VersionHandlerImpl/src/FileUtils.cs

```diff
@@ -656,6 +656,20 @@ public class RemoveAssetsResult {
                 return true;
             }
 
+            if (logger != null) {
+                logger.Log(
+                    String.Format(
+                        "Please ignore Unity error message like '{0}'.\n" +
```

Comment:
I would just say "Please ignore Unity error messages '{0}'"

Reply:
It is pretty counter intuitive that when Unity log an error and EDM4U says to ignore it without any explanation. 

I think this is better keep this long.

I have a feeling this is still not good enough since maybe some dev CI would expect no Error in their integration test.  But in that case, we can at least point them to the Unity bug report.

Reply:
It's not the length of the text, it's just a bit confusing to read.

How about `Please ignore Unity error messages similar to` ?